docs/Reference/API-Methods.md

```diff
@@ -4205,6 +4205,72 @@ data using `eea_sendRawTransaction`.
     The `PRIV` API methods are not enabled by default for JSON-RPC. Use the [`--rpc-http-api`](CLI/CLI-Syntax.md#rpc-http-api)
     or [`--rpc-ws-api`](CLI/CLI-Syntax.md#rpc-ws-api) options to enable the `PRIV` API methods.
 
+### priv_call
+
+Invokes a private contract function locally and does not change the state of the blockchain.
+
+For private contracts, `priv_call` is the equivalent to [`eth_call`](#eth_call).
+
+**Parameters**
+
+`data` - 32-byte [privacy Group ID](../Concepts/Privacy/Privacy-Groups.md).
+
+`object` - [Transaction call object](API-Objects.md#transaction-call-object).
+
+`quantity|tag` - Integer representing a block number or one of the string tags `latest`, `earliest`, 
+or `pending`, as described in [Block Parameter](../HowTo/Interact/APIs/Using-JSON-RPC-API.md#block-parameter).
+
+**Returns**
+
+`result` : `data` - Return value of the executed contract.
+
+!!! example
+    ```bash tab="curl HTTP"
+    curl -X POST --data '{"jsonrpc":"2.0","method":"priv_call","params":["tb8NVyQqZnHNegf/3mYsyB+HEud4SPWn90rz3GoskRw=", {"to":"0x69498dd54bd25aa0c886cf1f8b8ae0856d55ff13"}, "latest"],"id":1}' http://127.0.0.1:8545
```

Comment:
It makes more sense to show an example of executing a method in the contract.

```
{
    "jsonrpc": "2.0",
    "method": "priv_call",
    "params": [
        "tb8NVyQqZnHNegf/3mYsyB+HEud4SPWn90rz3GoskRw=",
        {
            "to": "0x69498dd54bd25aa0c886cf1f8b8ae0856d55ff13",
            "data": "0x3fa4f245"
        },
        "latest"
    ],
    "id": 1
}
```

Reply:
In the previous example: Calling the method in the smart contract identifired by the hash `0x3fa4f245 `

Reply:
Agreed - thanks for giving me this one